app/gui2/src/components/CircularMenu.vue

```diff
@@ -64,6 +70,47 @@ const emit = defineEmits<{
   }
 }
 
+.CircularMenuFull {
+  &:before {
+    content: '';
+    position: absolute;
+    clip-path: path(
+      evenodd,
+      'M0,52 A52,52 0,1,1 104,52 A52,52 0,1,1 0, 52 z m52,20 A20,20 0,1,1 52,32 20,20 0,1,1 52,72 z'
+    );
+    backdrop-filter: var(--blur-app-bg);
+    background: var(--color-app-bg);
+    width: 100%;
+    height: 100%;
+  }
+}
```

Comment:
Do we have to repeat properties in `.CircularMenuPartial` and `.CircularMenuFull`? @Frizi @somebody1234 any thoughts?

Reply:
Hmm, I just see more than a width: also height, backdrop-filter, background, content and position. Is there a reason we make it using pseudo-elements instead of normal element under the icons?

Reply:
For one, this was how it was implemented in the first place by @somebody1234. And my understanding is, that this allows the structure of the HTML to be simpler and has purely visual things in CSS instead of HTML.

Reply:
yeah. for me it's mostly a habit from the dashboard needing `backdrop-filter: blur` in a lot of places.

that said, it should be possible to use `:is(.Foo, .Bar)` for the properties that are common between the two.

alternatively you could do `.CircularMenu.partial` and `.CircularMenu.full` instead, and use `.CircularMenu` for the properties in common.
something like this might work:
```css
.CircularMenu {
  &:before {
    /* shared properties here */

    &.partial {
    }
    
    &.full {
    }
  }
}
```